docs/pages/access-controls/sso/github-sso.mdx

```diff
@@ -253,8 +236,60 @@ You will receive the details of your user session within the CLI.
   Extensions:         permit-port-forwarding, permit-pty, private-key-policy
 
 ```
+
 </ScopedBlock>
 
+## Step 4/4. Configure authentication preference
+
+In the previous step we signed in to Teleport using GitHub credentials by
+specifying GitHub as our auth type. By editing the `cluster_auth_preference`
+resource.
+
+Edit the existing `cluster_auth_preference` resource using `tctl`:
+
+```code
+$ tctl edit cap
```

Comment:
Note: `tctl edit` is not available in v11, so you'll have to change this section for the v11 backport.

Reply:
I believe what `tctl edit` does is something along these lines:

1. `tctl get cap > cap.yaml`
2. `$EDITOR cap.yaml`
3. `tctl create -f cap.yaml`